Extension/src/common.ts

```diff
@@ -339,6 +339,7 @@ export function resolveVariables(input: string | undefined, additionalEnvironmen
     let regexp: () => RegExp = () => /\$\{((env|config|workspaceFolder)(\.|:))?(.*?)\}/g;
     let ret: string = input;
     let cycleCache: Set<string> = new Set();
+    const delimiter: string = (process.platform === 'win32') ? ";" : ":";
```

Comment:
For better maintainability and since this code is repeated elsewhere, let's create a function in common.ts that returns delimiter based on a string input of the platform. This way, only the function needs to be updated if delimiter ever changes again.

Example
```
export function getEnvDelimiter(platform: string): string {
    return (platform === 'win32') ? ";" : ":";
}
```

Reply:
Where's the best place to put this function? Sorry, not super familiar with the codebase. In `Extension/src/common.ts` or somewhere else?

Reply:
Yes, I think Extension/src/common.ts.

Reply:
Since the delimiter won't change, how about this instead of a function?
```ts
// Platform-specific environment variable delimiter
export const envDelimiter: string = (process.platform === 'win32') ? ";" : ":";
```

Reply:
That seems good to me.